dotnetapp/dotnetapp.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
Can we assume that users will have MSBuild 15 on their machines?  My understanding is that it is associated with VS 2017, which is still just in RC. I wonder if using `ToolsVersion="14.0"` would work with both 14.0 and 15.0 and would thus be a safer choice.